docs/deprecations.rst

```diff
@@ -96,6 +73,29 @@ Removed features
 Deprecated features are only removed in major releases after an appropriate
 period of deprecation has passed.
 
+Image.__del__
+~~~~~~~~~~~~~
+
+*Removed in version 7.0.0.*
+
+Implicitly closing the image's underlying file in ``Image.__del__`` has been removed.
+Use a context manager or call ``Image.close()`` instead to close the file in a
+deterministic way.
+
+Deprecated:
```

Comment:
```suggestion
Previous method:
```
Should we say something other than 'Deprecated', since it's past that now?

Reply:
Good idea, done!